compiler/src/iree/compiler/DispatchCreation/SetEncoding.cpp

```diff
@@ -522,7 +551,29 @@ SmallVector<unsigned> getOperandsToPad(Operation *op) {
     return {};
   }
 
-  return {0, 1};
+  // Do not pad if the producer dispatch region contains an attention operation
+  // as that results in complicated multi-dimensional load/store access patterns
+  // which aren't well supported with padding.
+  // TODO(jtuyls): Try to generalize this by either providing better support in
+  // codegen or catching these load/store access patterns cases in a more
+  // general way.
```

Comment:
Can you file an issue and expand the context a bit more? I can't really understand what the issue is without running commands. The issue sounds not trivial, so I'd +1 on using an issue for tracking it.

Reply:
If the issue is already described in https://github.com/iree-org/iree/issues/20835, you can do `TODO(#20835): ...`